examples/nuget-build/Cargo.toml

```diff
@@ -14,4 +14,4 @@ winrt = { path = "../.." }
 "Microsoft.AI.MachineLearning" = "1.3.0"
 "Microsoft.Windows.SDK.Contracts" = "10.0.19041.1"
 
-[workspace]
\ No newline at end of file
+[workspace]
```

Comment:
Having the examples in their own workspace means that commands like `cargo test --all` and `cargo fmt --all` ignore the examples. This is undesirable, but adding them to the root workspace causes problems for `cargo winrt`.